travis_build.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-export SBT_OPTS="-server -Xms2G -Xmx2G -Xss10M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -XX:NewRatio=8 -XX:MaxPermSize=512M -XX:-UseGCOverheadLimit"
+# Maven Central and Bintray are unreachable over HTTPS
+if [[ "$TRAVIS_JDK_VERSION" == "openjdk6" ]]; then
+  SBT_OPTS="-Dsbt.override.build.repos=true -Dsbt.repository.config=./.sbtrepos"
```

Comment:
Thank you for this hack, didn't realise that one can override sbt repos this way...